lib/utils/testlog/log.go

```diff
@@ -36,10 +37,9 @@ func FailureOnly(t TestingInterface) utils.Logger {
 
 	// Register a cleanup callback which prints buf iff t has failed.
 	t.Cleanup(func() {
-		if !t.Failed() {
-			return
+		if t.Failed() || testing.Verbose() {
+			fmt.Fprintln(os.Stderr, buf.String())
```

Comment:
Found this bug when I was testing in verbose mode and wondering why there were no logs.